src/main/java/nl/novi/TechItEasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,116 @@
+package nl.novi.TechItEasy.controllers;
+
+import nl.novi.TechItEasy.Repositories.TelevisionRepository;
+import nl.novi.TechItEasy.exceptions.RecordNotFoundException;
+import nl.novi.TechItEasy.models.Television;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/televisions")
+public class TelevisionController {
+    @Autowired
+    TelevisionRepository televisionRepository;
+
+    @GetMapping
+    public ResponseEntity<Object> getAllTelevisions() {
+        if (televisionRepository.count() > 0) {
+            return new ResponseEntity<>(televisionRepository.findAll(), HttpStatus.OK);
+        } else {
+            throw new RecordNotFoundException("No televisions found");
+        }
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<Television> getTelevision(@PathVariable int id) {
+        if (televisionRepository.existsById(id)) {
```

Comment:
Nette oplossing volgens mij. In de uitwerking staat een variant die gebruik maakt van de "optional" functionaliteit, dat is misschien nog iets netter. Maar dit werkt goed.